express-static/public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Express Static</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <img
+      src="./images/f94b6f06289181924cffc9cc0bc0c23c.jpg"
```

Comment:
These need to be absolute paths, not relative paths. They need to start with `/` (so delete the leading `.`).

Confusing, yes! An absolute path means the browser will request the URL based on the host name in the address bar. A relative path means the browser will request the URL based on the entire URL in the address bar, not just the host name.

Reply:
Thank you for the explanation!!